server/py/services/api/api/endpoints/alert_activations.py

```diff
@@ -40,7 +40,7 @@ async def list_alert_activations(
     entity: Optional[str] = None,
     severity: Optional[
         list[Union[mlrun.common.schemas.alert.AlertSeverity, str]]
```

Comment:
@TomerShor we don't translate the given object to AlertConfig/AlertSeverity when listing alert activations. In terms of BC I was talking about another case. Imagine client has newer version of MLRun which support's new type of AlertSeverity. If user passes this new type in the filter along with other severities, we will fail fast. Do we want to do this instead of just ignoring it?

Reply:
If the client is newer then the server, this new severity won't be supported in the backend anyway.
IMO if we choose using the schemas as the query param types, then we should enforce them. If we don't want to enforce specific values then why use schemas at all?..

Reply:
@TomerShor I agree that we should have validation when storing data. But when filtering.. Especially if we allow the list here, I'd go with best-effort approach